.gitignore

```diff
@@ -6,3 +6,5 @@
 venv/
 .pytest_cache/
 **__pycache__/
+.venv/
```

Comment:
The `.venv/` entry is redundant because `venv/` is already listed on line 6. You can remove this line to avoid redundancy.